src/apps/adop_train.cpp

```diff
@@ -570,24 +586,24 @@ class NeuralTrainer
                 for (int i = 0; i < result.image_ids.size(); ++i)
                 {
                     // In average only write 10 images
-                    if (Random::sampleBool(std::min(1.0, 10.0 / loader_size)))
+                   // if (Random::sampleBool(std::min(1.0, 10.0 / loader_size)))
```

Comment:
Please revert, otherwise TB will gets thousands of images

Reply:
I will move the check to just catch TB. The goal was to write all test images every eval epoch to disc, not to clutter TB